src/components/Admin.vue

```diff
@@ -0,0 +1,248 @@
+<template>
+
+    <b-card class="text-center border-0">
+  
+      <h4 class="mb-4">Minting Transaction</h4>
+  
+      <b-form @submit.prevent="sendMintTransaction">
+  
+        <b-form-group class="mb-4">
+  
+          <div class="form-label-left">
+            <b-form-label for="public-key-input">Mint to</b-form-label>
+          </div>
+  
+          <!-- input field for the public key to mint to -->
+          <b-form-input
+            id="public-key-input"
+            name="public-key"
+            v-model="$v.form.mintingPublicKey.$model"
+            :state="validateState('mintingPublicKey')"
+            label="Minting to"
+            aria-describedby="address-help"
+            type="text"
+            placeholder="Enter public key"
+            required
+          ></b-form-input>
+  
+          <!-- describes the feedback area in the form if an invalid amount is entered -->
+          <b-form-invalid-feedback id="public-key-feedback" aria-describedby="address-help">
+            Address must be of the form e.g. usd1....xyz
+          </b-form-invalid-feedback>
+  
+        </b-form-group>
+  
+        <!-- field for the denomination of utxos to mint -->
+        <div class="form-row mb-4">
+          
+          <b-form-group class="col-md-6">
+  
+            <div class="form-label-left">
+              <b-form-label for="denomination-input">Value of UTXO</b-form-label>
+            </div>
+  
+            <!-- input field for the denomination of the token to mint -->
+            <b-form-input
+              id="denomination-input"
+              name="denomination"
+              v-model="$v.form.mintingValue.$model"
+              :state="validateState('mintingValue')"
+              label="Value of UTXO"
+              aria-describedby="number-format-help"
+              placeholder="Enter denomination"
+              required
+            ></b-form-input>
+  
+            <!-- describes the feedback that the form displays when a number is not in the correct format -->
+            <b-form-invalid-feedback id="denomination-input-feedback" aria-describedby="number-format-help">
+              Denomination must be in form 0.00
+            </b-form-invalid-feedback>
+  
+          </b-form-group>
+  
+          <!-- input field for the number of utxos to mint -->
+          <b-form-group class="col-md-6">
+  
+            <div class="form-label-left">
+              <b-form-label for="amount-input">Number of UTXOs to Mint</b-form-label>
+            </div>
+  
+            <!-- form input area for specifying the amount of utxos to mint -->
```

Comment:
nit: Ideally, the arrows provided here cannot even generate a negative number

Reply:
The regexp in numberFormatValidator below seems to require the string start with a digit. (Non-negative) should also check that this isn't a bug server side though as well.

Reply:
Not an issue for this PR but I'm a little worried that we are allowing unauthenticated posts to the mintTx endpoint - should probably rearchitect things at some point.

Reply:
opened #10 to capture this.